src/Helpers/SAMLUserGroupMapper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace SilverStripe\SAML\Helpers;
+
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\SAML\Services\SAMLConfiguration;
+use SilverStripe\Security\Group;
+use SilverStripe\Security\Member;
+
+class SAMLUserGroupMapper
+{
+    use Injectable;
+    use Configurable;
+
+    /**
+     * @var string
+     */
+    private static $group_claims_field;
+
+    /**
+     * @var array
+     */
+    private static $dependencies = [
+        'SAMLConfService' => '%$' . SAMLConfiguration::class,
+    ];
+
+    /**
+     * Check if group claims field is set and assigns member to group
+     *
+     * @param [] $attributes
+     * @param Member $member
+     * @return Member
+     */
+    public function map($attributes, $member): Member
+    {
+        $groups = $this->config()->get('group_claims_field');
```

Comment:
`config()` is a static method
```suggestion
        $groups = self::config()->get('group_claims_field');
```

Reply:
not a big deal but this style is valid and in our docs :) https://docs.silverstripe.org/en/5/developer_guides/configuration/configuration/#configuration-masks